Stratis.Bitcoin.Common/Stratis.Bitcoin.Common.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netstandard1.6</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;1705;IDE0008;</NoWarn>
```

Comment:
I can't see any warnings for CS1701, CS1702, CS1705. Also I don't think they need to be ignored should they show up.